File: mycollab-web/src/main/scala/com/mycollab/module/crm/event/CrmSettingEvent.scala
package com.mycollab.module.crm.event

import com.mycollab.events.ApplicationEvent

/**
  * @author MyCollab Ltd.
  * @since 5.0.3
  */
object CrmSettingEvent {

  class GotoNotificationSetting(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoCustomViewSetting(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

}

File: mycollab-web/src/main/java/com/mycollab/module/crm/event/CrmSettingEvent.kt
package com.mycollab.module.crm.event

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
object CrmSettingEvent {
    class GotoNotificationSetting(source: Any, val data: Any?) : ApplicationEvent(source)

    class GotoCustomViewSetting(source: Any, val data: Any?) : ApplicationEvent(source)
}